core/arch/arm/tee/entry_std.c

```diff
@@ -287,6 +288,8 @@ static void entry_invoke_command(struct thread_smc_args *smc_args,
 	struct tee_ta_session *s;
 	struct tee_ta_param param;
 
+	bm_timestamp();
```

Comment:
it's going to be mapped to specific file:line and function (the same as `addr2line` does)